uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/request-modal-details-toil.controller.js

```diff
@@ -3,13 +3,16 @@
 define([
   'common/lodash',
   'common/moment',
-  'leave-absences/shared/modules/controllers'
+  'leave-absences/shared/modules/controllers',
+  'common/services/crm-ang.service'
```

Comment:
the actual services/controllers/directives should be listed in the main modules' files

Reply:
Done.